src/main/java/xyz/nkomarn/harbor/listener/AfkListener.java

```diff
@@ -130,10 +143,9 @@ public AfkPlayer(Player player) {
          * @return true if the position changed
          */
         boolean changed() {
-            int hash = player.getLocation().hashCode();
-            boolean changed = hash != this.hash;
-            this.hash = hash;
-            return changed;
+            Location prev = location;
+            location = player.getEyeLocation();
+            return !prev.equals(location);
```

Comment:
Why did this change to Location instead of it's hash?

Reply:
Dont know. Maybe we just just merge this and I will create another Merge request and fix based on my recommendations.
We are already discussing on this PR for a month now and I am getting slightly tired.

Reply:
I was trying to test something and forgot to revert it, it was part of trying to troubleshoot the "everyone is afk" issue, using a debugger